freedomofpress/securedrop
# Update base SD docker app container

## Status

Ready for review

## Description of Changes

Fixes #3510 

Changes proposed in this pull request:

## Testing

How should the reviewer test this PR?

(FYI need push rights on `quay.io`)

* cd into `molecule/builder`
* run `make build-container`
* edit `push.sh` and replace an arbitrary string for the `DATE_STR` variable up top
* run `make push-container`
* cd back to core dir (`../../`)
* run `make build-debs` and confirm no errors

Go and clean up the image (should be same label as the `DATE_STR` you set) you pushed on quay. going to have to do this thru the webui

## Deployment

None - only affects debian package build process
